task/sql-tasks.js

```diff
@@ -177,7 +278,14 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+	        COUNT(ProductName) AS "TotalOfCurrentProducts",
+	        SUM(Discontinued) AS "TotalOfDiscontinuedProducts"

```

Comment:
use count when you need to count recrods to show intent explicitly

Reply:
but in this case it's even better to do this 

```
select
(select count()...) as 'T'
(select count()...) as 'TT'
```

notice that from clause is optional